draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -604,10 +610,19 @@ to a stub resolver for an SVCB record query:
    the results in the Additional section.
 
 3. Otherwise, select an AliasForm record at random, and resolve A, AAAA,
-   and SVCB records for
-   the SvcDomainName.  If the SVCB record does not exist, add the A and AAAA
-   records to the Additional section.  Otherwise, go to step 1,
-   subject to loop detection heuristics.
+   and SVCB records for the SvcDomainName.  If the SVCB record does not exist,
+   add all records from the A and AAAA responses to the Additional section.
+   Otherwise, go to step 1, subject to loop detection heuristics.
+
+In steps 2 and 3, "resolve" means the resolver's ordinary iterative
```

Comment:
"iterative" or "recursive" ?  I typically see the latter used.

Reply:
I think you're right.  Changed to "recursive".